f5-bigiq-generate-cert-with-SAN/generate-self-signed-cert

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+IP=$(echo "$1" | grep -E -o "^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$")
+DAYS="$2"
+KEY_LENGTH="$3"
+
+if [ ! "$IP" ] || [ ! "$DAYS" ] || [ ! "$KEY_LENGTH" ]
+then
+    echo "Usage: generate-ip-cert.sh 127.0.0.1 365 4096"
+    exit 1
+fi
+
+echo "[req]
+default_bits  = 3072
+distinguished_name = req_distinguished_name
+req_extensions = req_ext
+x509_extensions = v3_req
+prompt = no
+
+[req_distinguished_name]
+countryName = XX
+stateOrProvinceName = N/A
+localityName = N/A
+organizationName = Self-signed certificate
+commonName = $IP: Self-signed certificate
+
+[req_ext]
+subjectAltName = @alt_names
+
+[v3_req]
+subjectAltName = @alt_names
+
+[alt_names]
+IP.1 = $IP
+" > san.cnf
+
+openssl req -x509 -nodes -days "$DAYS" -newkey rsa:"$KEY_LENGTH" -keyout key.pem -out cert.pem -config san.cnf
+rm san.cnf
+
+# backup the original certificate and key
+mv /config/httpd/conf/ssl.crt/server.crt /config/httpd/conf/ssl.crt/server.crt.default
+mv /config/httpd/conf/ssl.key/server.key /config/httpd/conf/ssl.key/server.key.default
+
+# rename the newly generated key and crt
+mv cert.pem /config/httpd/conf/ssl.crt/server.crt
+mv key.pem /config/httpd/conf/ssl.key/server.key
+
+# restart webd
+tmsh restart sys service webd
```

Comment:
Could you add a step which checks when the webd service is restarted.

e.g. with other scripts:
        echo "Restart restjavad..."
        bigstart restart restjavad &
        restartProc=$!
        wait $restartProc
        sleep 5
        echo "Restart restnoded..."
        bigstart restart restnoded &
        restartProc=$!
        wait $restartProc
        echo "Finished restarting services"